### PR TITLE
Release v3.0.3

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: CHANGE-NOTES.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## v3.0.3
+- Enabled Ctrl + mouse click to work as a right-click for Mac users
```
Comment (vzmerr) [inline CHANGE-NOTES.md:6]:
```suggestion
- Enabled Ctrl + left-click to work as a right-click for Mac users, for displaying the machete actions on a branch.
```
left click can happen both on the Touch Bar and with mouse.

Comment (LukaszKontowski) [reply CHANGE-NOTES.md:6]: 👍

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -411,7 +411,7 @@ You need to download the [IntelliJ signer CLI](https://github.com/JetBrains/mark
 
 ```shell
 java -jar marketplace-zip-signer-cli.jar sign\
-  -in "git-machete-intellij-plugin-version-SNAPSHOT+git.number.zip"\
+  -in "$(./gradlew -q printPluginZipPath)"\
```
Comment (vzmerr) [inline CONTRIBUTING.md:414]:
```suggestion
  -in "$(./gradlew -q $printPluginZipPath)"\
```
isn't print plugin zip path a variable?

File: .circleci/config.yml [hunk 3]
```diff
@@ -229,17 +228,17 @@ jobs:
...
                   corresponding_milestone_title=v$(extract_version_from_current_wd)
                   tag=$corresponding_milestone_title
                   change_notes=$(./gradlew getChangelog --no-header -q)
-                  hub release create \
-                    --attach build/distributions/git-machete-intellij-plugin-*.zip \
-                    --message "$tag"$'\n\n'"$change_notes" \
-                    $tag
+                  gh release create \
+                    $tag \
+                    "$(./gradlew -q printSignedPluginZipPath)" \
```
Comment (vzmerr) [inline .circleci/config.yml:240]:
```suggestion
                    "$(./gradlew -q $printSignedPluginZipPath)" \
```
isn't `printSignedPluginZipPath` a variable?

File: docs/features.md [hunk 4]
```diff
@@ -39,6 +39,8 @@ The current branch is underlined in a branch layout.
 
 Git Machete can display commits belonging to the branches.
 It displays only the commits that are unique to the selected branch (i.e. have never been a tip of a different branch).
+Commits are displayed above their branch name, starting from the oldest ones
```
Comment (vzmerr) [inline docs/features.md:42]:
```suggestion
Commits are displayed above their branch name, starting from the oldest ones.
```

File: docs/features.md [hunk 5]
```diff
@@ -39,6 +39,8 @@ The current branch is underlined in a branch layout.
...
+(the newest commit is displayed just above the branch name).
```
Comment (vzmerr) [inline docs/features.md:43]:
```suggestion
Commits are displayed above their branch name, starting from the oldest ones, with the newest commit displayed just above the branch name.
```

Comment (PawelLipski) [reply docs/features.md:43]: (suggestions generally to be applied on a PR to be merged to `develop`... if you could tend to that @LukaszKontowski

Comment (LukaszKontowski) [reply docs/features.md:43]: Sure

**Comment by LukaszKontowski** [comment]
Suggested changes are in this PR: https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1197

File: CHANGE-NOTES.md (hunk 1 above)
Comment (mkondratek) [inline CHANGE-NOTES.md:6]: also for consistency we refer mac with `macOS`

> No, they are essentially the same thing — just named differently. In fact, three terms were used at different times with reference to Apple’s operating system: Mac OS X, OS X, and macOS. Mac OS X was the official naming through version 10.7, from 2001 to 2011. In the next four years, the OS X names were used. Finally, Apple shifted to "macOS" with the release of macOS High Sierra in 2016. The latter helped standardize the naming of Apple’s operating systems — macOS, iOS, tvOS, iPadOS, etc. 

~ https://setapp.com/how-to/full-list-of-all-macos-versions

let's change it here and within CONTRIBUTING.md#Plugin signing as part of the CI publish process through the Gradle Plugin

Comment (LukaszKontowski) [reply CHANGE-NOTES.md:6]: Done as suggested in https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1197

**Review by LukaszKontowski — APPROVED** [review]

**Review by vzmerr — APPROVED** [review]

**Review by mkondratek — APPROVED** [review]

**Pull request merged**